src/DataAccess/Repositories/Redis/RedisEdgeRepository.cs

```diff
@@ -270,5 +270,54 @@ public async Task<(bool, EdgeModel?)> CheckIfIdExists(string id)
                 return (false, matchedEdge);
             }
         }
+
+        /// <summary>
+        /// Return online status and LastUpdatedTime of edge by cloud Id.
+        /// </summary>
+        /// <param name="edgeId"></param>
+        /// <returns>Return bool of isOnline and dateTime of lastUpdatedTime of the edgeModel</returns>
+        [Obsolete]

```

Comment:
Should this function be `Obsolete`? This tag means that the function is unused and will shortly be removed